src/encoded/static/components/publication.js

```diff
@@ -26,7 +73,11 @@ const PublicationComponent = (props, reactContext) => {
     ];
 
     const crumbsReleased = (context.status === 'released');
-
+    const accessions = !context.datasets ? '' : context.datasets.reduce((dataset1, dataset2) => `${dataset1}&accession=${dataset2.accession}`, '');
```

Comment:
Maybe just change to `context.datasets` without the `!` so you can reverse the conditions? I had to read over this twice to realize it wasn’t a mistake.

Reply:
@forresttanaka I will reverse it.